lms/templates/instructor/instructor_dashboard_2/sensitive_data_download_msg.txt

```diff
@@ -0,0 +1,5 @@
+<%!
```

Comment:
and this is the template that we did not want to duplicate from CAM-12 and we expect to exist in the repo after the merge/rebase is done?